qlib/SftpPoller.qm

```diff
@@ -267,6 +275,8 @@ public namespace SftpPoller {
             - \c log_debug: a @ref closure "closure" or @ref call_reference "call reference" for logging verbose debgugging information; must accept a single string giving the log message
             - \c start_thread: (required when imported into a context where @ref Qore::PO_NO_THREAD_CONTROL is set) a @ref closure "closure" or @ref call_reference "call reference" for starting threads; must return the integer thread ID (if not set then @ref background will be used)
             - \c sleep: (required when imported into a context where @ref Qore::PO_NO_PROCESS_CONTROL is set) a @ref closure "closure" or @ref call_reference "call reference" to use instead of @ref Qore::sleep() (if not set then @ref Qore::sleep() will be used)
+            - \c writable: check if path(s) is/are writable  (default: False \c see also "check_file" option)
```

Comment:
\c see is incorrect - please check the documentation to verify your formatting - you probably want to use see also the \c "check_file" option instead